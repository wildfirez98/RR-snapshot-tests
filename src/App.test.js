import { render, screen } from '@testing-library/react';
import App from './App';
import renderer from 'react-test-renderer';

test('Renders a snapshot', () => {
  const tree = renderer.create(<App />).toJSON()
  console.log(tree)
  expect(tree).toMatchSnapshot()
})
