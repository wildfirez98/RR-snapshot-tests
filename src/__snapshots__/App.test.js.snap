// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a snapshot 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p>
      Edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
    <a
      className="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React
    </a>
    <h3>
      Testing Snapshots Out!
    </h3>
    <h2>
      Another Update!
    </h2>
  </header>
</div>
`;
